skyrl-tx/.gitignore

```diff
@@ -76,3 +76,13 @@ Thumbs.db
 # Local artifacts
 tinker.db
 uv.lock
+
+# Alembic - don't track migrations or pycache
+tx/tinker/alembic/__pycache__/
+tx/tinker/alembic/versions/
```

Comment:
The standard practice is that all the migrations get committed to the repo (so everybody can run them), so I'll remove this line :)